src/config/colors.js

```diff
@@ -0,0 +1,9 @@
+export const Blue = {
```

Comment:
Why not in css file?

Reply:
'colors' should be reused between many files. Overriding mui-components is much easier using the 'sx' prop, than creating a ccs and importing it, and making sure in it to override relevant classes as seen on the browser (need to manually search the dom).